docs/content/setup/vmware.md

```diff
@@ -9,22 +9,19 @@ tags: ["setup", "vmware"]
 
 ## Deployment
 
-1. Deploy the Infection Monkey OVA by choosing Deploy OVF Template and follow the wizard instructions. *Note: make sure port 5000 and 5001 on the machine are accessible for inbound TCP traffic.*
+1. Deploy the Infection Monkey OVA by choosing **Deploy OVF Template** and following the wizard instructions. *Note: make sure ports 5000 and 5001 on the machine are accessible for inbound TCP traffic.*
 2. Turn on the Infection Monkey VM.
 3. Log in to the machine with the following credentials:
    1. Username: **monkeyuser**
    2. Password: **Noon.Earth.Always**
-4. It's recommended to change the machine passwords by running the following commands: `sudo passwd monkeyuser`, `sudo passwd root`.
+4. It's recommended you change the machine passwords by running the following commands: `sudo passwd monkeyuser`, `sudo passwd root`.
 
 ## OVA network modes
 
-The OVA can be used in one of two modes:
+You can use the OVA in one of two modes:
 
-1. In a network with DHCP configured. In this case, the Monkey Island will automatically query and receive an IP address from the network.
-1. With a static IP address.
-
-    In this case, you should login to the VM console with
-username `root` and password `G3aJ9szrvkxTmfAG`. After logging in, edit the interfaces file. You can do that by writing the following command in the prompt:
+1. In a network with the DHCP configured— In this case, the Monkey Island will automatically query and receive an IP address from the network.
+2. With a static IP address — For this case, you should login to the VM console with the username `root` and the password `G3aJ9szrvkxTmfAG`. After logging in, edit the interfaces file by writing the following command in the prompt:
```

Comment:
@MarketingYeti Since this documentation is written in markdown, we can take advantage of some markdown features.

Because lists change, and it's a pain to go through and renumber lists, when a markdown list is rendered, it will replace all the `1.` lines with the appropriate number. A list that looks like

```
1. item 1
1. item 2
1. item 3
```

Will be rendered as 
```
1. item 1
2. item 2
3. item 3
```
We should leave the numbering in this list the way it was. We should also fix the list starting on line 12 so that all list numbers are `1.`. Ditto for the rest of the documentation.